cli/src/commands/sidekick.rs

```diff
@@ -465,4 +470,17 @@
 
 		return Ok(());
 	}
+
+	pub async fn get_api_endpoint(&self) -> Result<()> {
+		let api_endpoint = global_config::read_project(|x| x.cluster.api_endpoint.clone()).await?;
+
+		struct_fmt::print(
+			&Format::Json,
+			&SideKickResponse::Ok(json!({
+				"api_endpoint": api_endpoint
+			})),
+		)?;
+
+		Ok(())
+	}
```

Comment:
I'm curious if we should put this in one single command that returns everything about the state of the CLI. From it, you might be able to get the CLI version, the api endpoint, and any other needed data. That way we can just add to that. I'm not sure if this has ramifications for backwards compatibility though.

Reply:
Oh, yep! I was thinking the same. Let me re-do this.